modules/mono/editor/bindings_generator.cpp

```diff
@@ -365,7 +365,7 @@ String BindingsGenerator::bbcode_to_xml(const String &p_bbcode, const TypeInterf
 				xml_output.append("</c>");
 			} else if (link_tag == "enum") {
 				StringName search_cname = !target_itype ? target_cname :
-														  StringName(target_itype->name + "." + (String)target_cname);
+															StringName(target_itype->name + "." + (String)target_cname);
```

Comment:
```suggestion
						StringName(target_itype->name + "." + (String)target_cname);
```

Why does clang-format insist on alignment (...and with what, btw)? I think we should instead just make continuation lines have the same indent level as the previous line, plus two, like the GDScript style guide says.

Reply:
We can't fine tune that AFAIK. It's already configured to use two tabs for continuation, but clang-format insists on adding spaces for some weird alignment.

Reply:
I looked at everything I could think of, I couldn't find any way to turn this behavior off and make it behave like regular continuation lines. I opened [this bug report](https://bugs.llvm.org/show_bug.cgi?id=48736), hopefully we will be able to do this in the future.

Reply:
For the reference, there's an `AlignOperands` option which mentions ternary operators: https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Notably:

> OAS_DontAlign (in configuration: DontAlign) Do not align operands of binary and ternary expressions. The wrapped lines are indented ContinuationIndentWidth spaces from the start of the line.

But when I tried it it didn't seem to solve this case. It does impact other stuff which we might want to assess as beneficial, but not this issue with force-aligning the last branch of a ternary operator to some arbitrary column.

Reply:
> I opened this bug report, hopefully we will be able to do this in the future.

If you can, I'd suggest attaching a minimal reproduction project to that bug report with a `main.cpp` file and a `.clang-format` config (could be Godot's, or a more minimal one). You can use it to show both what you'd like the formatting to be like (enclosed in `/* clang-format off */` and `/* clang-format on */`), and what it ends up being when applying clang-format.